cogs/moderator.py

```diff
@@ -65,7 +65,8 @@ def cog_unload(self):
     @command(help="Verify a mentioned user")
     @bot_chat(bot_channels)
     @has_role_plus(moderator)
-    async def verify(self, ctx: Context, member):  # parameter only for pretty help.
+    # parameter only for pretty help.
+    async def verify(self, ctx: Context, member):
```

Comment:
Do not change unrelated lines.

```suggestion
    async def verify(self, ctx: Context, member):  # parameter only for pretty help.
```